UnityProject/Assets/Tests/Scenes/SceneTest.cs

```diff
@@ -49,27 +49,7 @@ public void Setup()
 				MapSaver.MapSaver.CodeClass.ThisCodeClass.Reset();
 				MapSaver.MapSaver.MapData mapData = JsonConvert.DeserializeObject<MapSaver.MapSaver.MapData>(AccessFile.Load(Data.File, FolderType.Maps));
 				List<IEnumerator> PreviousLevels = new List<IEnumerator>();
-				var Imnum = MapLoader.ServerLoadMap(Vector3.zero, Vector3.zero, mapData);
-				bool Loop = true;
-				while (Loop && PreviousLevels.Count == 0)
-				{
-					if ( Imnum.Current is IEnumerator)
-					{
-						PreviousLevels.Add(Imnum);
-						Imnum = (IEnumerator) Imnum.Current;
-					}
-
-					Loop = Imnum.MoveNext();
-					if (Loop == false)
-					{
-						if (PreviousLevels.Count > 0)
-						{
-							Imnum = PreviousLevels[PreviousLevels.Count - 1];
-							PreviousLevels.RemoveAt(PreviousLevels.Count - 1);
-							Loop = Imnum.MoveNext();
-						}
-					}
-				}
+				_ = MapLoader.ServerLoadMap(Vector3.zero, Vector3.zero, mapData);
```

Comment:
this needs to be awaited, since tests run immediately after this, it can't be run in parallel

Reply:
this technically is awaited because `_` stops the code until the function is done

Reply:
that is not how it works. ``_`` is there just so you can fire and forget without getting a warning from the compiler for not using the returned value.